Algorithmia/client.py

```diff
@@ -33,11 +33,12 @@ def algo(self, algoRef):
         return Algorithm(self, algoRef)
 
     def file(self, dataUrl):
-        if dataUrl.startswith('file://'): return LocalDataFile(self, dataUrl)
+        if dataUrl.startswith('local://'): return LocalDataFile(self, dataUrl)
```

Comment:
What was the imputus to change the `file://` prefix to `local://`? `file://` is somewhat a standard name for operations like this in other applications:
https://stackoverflow.com/questions/55143141/what-does-file-prefix-mean-when-reading-a-file

Reply:
You're right - bad call on my part and I will change it.  Thanks!